Description: systemd: Configure ntpviz
 - Configure ntpviz to match the cron.d jobs
 - ntpd.service is called ntpsec.service in this package
Forwarded: not-needed
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2019-09-23
--- a/etc/ntpviz-daily.service
+++ b/etc/ntpviz-daily.service
@@ -1,9 +1,10 @@
 [Unit]
 Documentation=man:ntpviz(1)
 Description=Graph daily information for NTP
-Requisite=ntpd.service
+Requisite=ntpsec.service
 
 [Service]
-Type=simple
+Type=oneshot
 IOSchedulingClass=idle
-ExecStart=@BINDIR@/ntpviz -w l -p 1 -o /var/www/localhost/htdocs/day
+ExecStart=@BINDIR@/ntpviz -p 1 -d /var/log/ntpsec -o /var/lib/ntpsec/ntpviz/day @/etc/ntpviz/options
+ExecStartPost=/usr/bin/find /var/lib/ntpsec/ntpviz/day -type f -mtime +1 -delete
--- a/etc/ntpviz-weekly.service
+++ b/etc/ntpviz-weekly.service
@@ -1,9 +1,10 @@
 [Unit]
 Documentation=man:ntpviz(1)
 Description=Graph weekly information for NTP
-Requisite=ntpd.service
+Requisite=ntpsec.service
 
 [Service]
-Type=simple
+Type=oneshot
 IOSchedulingClass=idle
-ExecStart=@BINDIR@/ntpviz -w l -p 7 -o /var/www/localhost/htdocs/week
+ExecStart=@BINDIR@/ntpviz -p 7 -d /var/log/ntpsec -o /var/lib/ntpsec/ntpviz/week @/etc/ntpviz/options
+ExecStartPost=/usr/bin/find /var/lib/ntpsec/ntpviz/week -type f -mtime +7 -delete
--- a/etc/ntplogtemp.timer
+++ b/etc/ntplogtemp.timer
@@ -7,4 +7,4 @@
 Persistent=true
 
 [Install]
-WantedBy=ntpd.service
+WantedBy=ntpsec.service
--- a/etc/ntpviz-daily.timer
+++ b/etc/ntpviz-daily.timer
@@ -6,4 +6,4 @@
 Persistent=true
 
 [Install]
-WantedBy=ntpd.service
+WantedBy=ntpsec.service
--- a/etc/ntpviz-weekly.timer
+++ b/etc/ntpviz-weekly.timer
@@ -6,4 +6,4 @@
 Persistent=true
 
 [Install]
-WantedBy=ntpd.service
+WantedBy=ntpsec.service
